config/stage.json

```diff
@@ -72,6 +72,7 @@
 		"post-editor/html-toolbar": true,
 		"post-editor/insert-menu": true,
 		"post-editor/image-editor": true,
+		"post-editor/mentions": true,
```

Comment:
We generally have stage mirror prod settings. If you'd like to open this up to more testing environments, I would recommend wpcalypso and horizon instead.